src/miral/x11_support.cpp

```diff
@@ -42,6 +42,11 @@ void miral::X11Support::operator()(mir::Server& server) const
         mo::x11_display_opt,
         "Enable X11 support", mir::OptionType::null);
 
+    server.add_configuration_option(
+        mo::x11_scale_opt,
+        "The scale to assume X11 apps use. If unset uses the value of GDK_SCALE or 1. Can be fractional",
+        mir::OptionType::string);
```

Comment:
(Out of scope for this PR)

Maybe we should plumb through a `float` configuration type.